packages/vats/scripts/restart-vats.js

```diff
@@ -0,0 +1,23 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
+export const defaultProposalBuilder = async () => {
+  // An includelist isn't necessary because the collections are known to be complete (tested in test-vaults-upgrade.js)
+  const skip = [
+    // can be replaced instead of upgraded
+    'auctioneer',
+    'feeDistributor',
+    // skip so vaultManager can have prices upon restart; these have been tested as restartable
+    'scaledPriceAuthority-ATOM',
+  ];
```

Comment:
here's hoping i find time to confirm that we have written details on how these replacements work... or that the relevant folks know we're trusting to luck a bit.